.github/workflows/arc-prerelease-deployment.test.yml

```diff
@@ -0,0 +1,46 @@
+name: arc prerelease deployment test
+
+on:
+  release:
+    types: [prereleased]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: npm ci
+        working-directory: ./examples/arc-prerelease-deployment-test
```

Comment:
I'll give this a deeper look later, but one thing I'd like is if we can use create-remix instead of just using the pre-built example directories. That way we get more confidence out of these tests that the generators work.

Reply:
agreed, just need to tweak `create-remix` a bit to allow for easier app creation

Reply:
alllllrighty @kentcdodds, just spent the last ~4 hours just to realize the default npm version in GitHub actions is 6... but... each app is created on the fly now